src/test/java/eu/dissco/core/translator/terms/specimen/SpecimenNameTest.java

```diff
@@ -0,0 +1,66 @@
+package eu.dissco.core.translator.terms.specimen;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.dissco.core.translator.terms.specimen.SpecimenName;
+import org.gbif.dwc.ArchiveField;
+import org.gbif.dwc.ArchiveFile;
+import org.gbif.dwc.record.Record;
+import org.gbif.dwc.terms.DwcTerm;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class SpecimenNameTest {
+
+  private final SpecimenName specimenName = new SpecimenName();
+  @Mock
+  private ArchiveFile archiveFile;
+  @Mock
+  private Record rec;
+
+  @Test
+  void testRetrieveFromDWCA() {
+    // Given
+    var specimenNameString = "SpecimenName";
+    var archiveField = new ArchiveField(0, DwcTerm.scientificName);
+    given(archiveFile.getField("dwc:scientificName")).willReturn(archiveField);
+    given(rec.value(archiveField.getTerm())).willReturn(specimenNameString);
+
+    // When
+    var result = specimenName.retrieveFromDWCA(archiveFile, rec);
+
+    // Then
+    assertThat(result).isEqualTo(specimenNameString);
```

Comment:
Why is the specimenNameString a static string?

Reply:
Oh wait, is it because it is a test

Reply:
Yes could be any string, all that matters is that it comes out equal